docs/tutorials/quickstart.rst

```diff
@@ -0,0 +1,109 @@
+Quickstart
+==========
+
+This tutorial gets you up-and-running quickly with NAPALM in a local virtual environment so you can see it in action.
+
+.. note::  This tutorial does not cover fully automated configuration management (e.g., using NAPALM in conjunction with Ansible, Chef, Salt, etc.).  We hope that tutorials for these tools will be contributed soon so that you can evaluate the options for your particular environment.
+
+Requirements
+------------
+
+You'll need a few tools:
+
+* Python
+* `pip <https://pip.pypa.io/en/stable/installing/>`_: The PyPA recommended tool for installing Python packages
+* `virtualenv <http://docs.python-guide.org/en/latest/dev/virtualenvs/>`_: a tool to create isolated Python environments
+* `VirtualBox <https://www.virtualbox.org/>`_: a software virtualization tool
+* `Vagrant <https://www.vagrantup.com/downloads.html>`_: a command line utility for managing the lifecycle of virtual machines
+
+As the focus of this tutorial is NAPALM, we don't even scratch the surface of these tools.  If you're not familiar with them, please do some research [#f1]_ as they will be an important part of your development/ops toolkit.
+
+Installation
+------------
+
+If you haven't already installed NAPALM, you can run the code from a virtual environment in the root directory::
+
+    $ virtualenv venv                        # create environment
+    $ source venv/bin/activate               # start environment
+    $ sudo pip install -r requirements.txt   # install dependencies
+    $ python setup.py install                # install NAPALM
```

Comment:
Don't use this method. Use pip instead, please. There are some caveats when installing the new modular code with this mechanism.